### PR TITLE
Update Dashboard (CU-86du23zth)

### PR DESCRIPTION
**Pull request by Darrickg**
Test Plan:
- [x] Tested locally
- [x] Tested on dev
- [x] Integration test cases for new client, edit client, and edit button

- new button has been split into a new task
- testing gateways has been split into a new task

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86du23zth Make the buttons dashboard more useful](https://app.clickup.com/t/86du23zth)

**Review by micahdbak — APPROVED** [review]
👍 

feel free to merge without test cases

**Comment by Darrickg** [comment]
@johnny-brave finally done, you can test!

File: server/mustache-templates/clientPage.mst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 
 <head>
-    <title>Brave Chatbot Dashboard</title>
+    <title>Brave Sensor Dashboard</title>
```
Comment (Nakul727) [inline server/mustache-templates/clientPage.mst:5]: * Buttons

File: server/mustache-templates/newGateway.mst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        {{> nav}}
+        <div class="content-area">
+            <div class="content-wrapper">
+                <h4 class="no-locations">Add a Gateway</h4>
+                <h6> Please fill in the form below. Any fields not seen here will be filled in with default values on submission and can be modified by clicking the "edit" button on the next screen.</h6>
+                <br>
+                <form class="needs-validation" action="/gateways" method="POST">
+
+                    <div class="form-group row justify-content-start row-no-gutters">
+                        <label for="gatewayId" class="col-sm-3 col-form-label"></label>
```
Comment (Nakul727) [inline server/mustache-templates/newGateway.mst:26]: Add label for gateway id

**Review by Nakul727 — CHANGES_REQUESTED** [review]
Have tested the dashboard to my best and everything works perfectly! Good job 👍Just a few minor changes I have annotated for change. 

Along with that a small cases of error handling should be added if invalid gateway id (not uuid) is inputed. It gives a 500 error, shows blank screen but logs the error in server. Maybe add check to handle this.

Otherwise good from my side, everything works!!

File: server/mustache-templates/clientPage.mst (hunk 1 above)
Comment (Nakul727) [reply server/mustache-templates/clientPage.mst:5]: Also check title headers for the other pages

File: server/mustache-templates/newGateway.mst (hunk 2 above)
Comment (Darrickg) [reply server/mustache-templates/newGateway.mst]: fixed in latest push

**Review by Nakul727 — APPROVED** [review]
Looks good to me, amazing work! Wait on @johnny-brave to review and test and should be ready to merge :))

**Review by johnny-brave — APPROVED** [review]
Very impressive work! Thanks for taking on such a big task, this will be really helpful for future provisioning. I've tested all the functionalities of the dashboard with real dev devices and everything is working correctly. The only thing I noticed is that this says 'Radar Particle Core ID' when it should say 'Serial Number'. Please make that change
image
 Other than that, If Nakul approves of the code, then it's ready to merge.

**Pull request merged**